app/src/controllers/log/sagas.js

```diff
@@ -195,10 +197,16 @@ function* fetchHistoryItemData() {
 
 function* fetchLogPageData({ meta = {} }) {
   const isPathNameChanged = yield select(pathnameChangedSelector);
+  const logItem = yield select(activeLogSelector);
   yield put({ type: CLEAR_NESTED_STEPS });
   if (meta.refresh) {
     const offset = yield select(logPageOffsetSelector);
-    yield all([put(fetchTestItemsAction({ offset })), call(fetchLogs)]);
+    yield all([
+      put(fetchTestItemsAction({ offset })),
+      put(fetchLogPageStackTrace(logItem)),
+      put(fetchFirstAttachmentsAction()),
```

Comment:
Could you please explain why you added these two fetches?
 `put(fetchLogPageStackTrace(logItem))`
 `put(fetchFirstAttachmentsAction())`

Reply:
Refresh does not work without them on the Stack trace tab and Attachment tab